lib/modules/styleTweaks.js

```diff
@@ -111,6 +111,12 @@ modules['styleTweaks'] = {
 			description: 'Add an icon in the omnibar (where the page address is written) to disable/enable current subreddit style.',
 			noconfig: true // only show for chrome
 		},
+		disableSubredditStyle: {
+			type: 'boolean',
+			value: false,
+			description: 'Always remove subreddit style.'
```

Comment:
I personally don't think this is enough. People might be really stupid.

Use a hover similar to np warnings when enabling?